src/index.ts

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/env node
-import chalk from "chalk";
-import clear from "clear";
-import figlet from "figlet";
-import program from "commander";
-import "./commands/init";
-clear();
+import chalk from 'chalk'
+import clear from 'clear'
+import figlet from 'figlet'
+import program from 'commander'
+import './commands/init'
+clear()
 console.log(
-  chalk.red(figlet.textSync("CATS-CLI", { horizontalLayout: "full" }))
-);
-program.version("0.0.1").description("An example CLI for ordering pizza's");
+  chalk.red(figlet.textSync('CATS-CLI', { horizontalLayout: 'full' }))
+)
+program.version('0.0.1').description("An example CLI for ordering pizza's")
```

Comment:
Maybe we should the change the description here and get the version ?

Reply:
actually haven't found a way to get version from package json since `/cats/src'. 'rootDir' is expected to contain all source files`

Reply:
changed the description though

Reply:
I'm not sure but this should be working : `const version = process.env.npm_package_version;`